app/src/main/java/org/thoughtcrime/securesms/conversation/v2/ConversationActivityV2.kt

```diff
@@ -1310,7 +1326,7 @@ class ConversationActivityV2 : PassphraseRequiredActionBarActivity(), InputBarDe
     }
 
     override fun resendMessage(messages: Set<MessageRecord>) {
-        messages.forEach { messageRecord ->
+        messages.iterator().forEach { messageRecord ->
```

Comment:
What is the significance of calling `collection.iterator().forEach` instead of `collection.forEach`? 🤔

Reply:
just writing here to keep all in one place, but the `.iterator()` was required to satisfy a lint error for API 23 when running `./gradlew build`